src/main/webapp/app/overview/submission-result-status.component.html

```diff
@@ -27,6 +27,12 @@
             <span class="text-body-secondary" *ngIf="!notSubmitted && studentParticipation?.initializationState === InitializationState.FINISHED">{{
                 'artemisApp.courseOverview.exerciseList.userSubmitted' | artemisTranslate
             }}</span>
+            <span
+                class="text-body-secondary"
+                *ngIf="studentParticipation?.initializationState === InitializationState.FINISHED && !studentParticipation?.submissions?.[0]?.plagiarismDetected"
+            >
+                {{ 'artemisApp.courseOverview.exerciseList.userSubmitted' | artemisTranslate }}
+            </span>
```

Comment:
Agreed with Jakub, that this need to be removed.

Reply:
Removed